src/Command/RunCommand.php

```diff
@@ -29,14 +29,18 @@ protected function configure(): void
     {
         $this->addOption('skip-theme-compile', null, InputOption::VALUE_OPTIONAL, 'Skip theme compile (should be used when the Theme has been compiled before in the CI/CD)', false);
         $this->addOption('skip-asset-install', null, InputOption::VALUE_OPTIONAL, 'Skip asset install (should be used when the Assets has been copied before in the CI/CD)', false);
+        $this->addOption('timeout', null, InputOption::VALUE_OPTIONAL, 'Set script execution timeout (is seconds). Set to null to disable timeout', 60);
```

Comment:
maybe higher, like 2 minutes? 🤔 wdym?

Reply:
I personally disable it completely using `---timeout=null`
60s is the symfony default. When increasing, then maybe 5min

Reply:
5 min sounds good 👍